usecase/src/test/java/org/mobilitydata/gtfsvalidator/usecase/ParseAllExecutionParameterTest.java

```diff
@@ -0,0 +1,50 @@
+package org.mobilitydata.gtfsvalidator.usecase;
+
+import org.junit.jupiter.api.Test;
+import org.mobilitydata.gtfsvalidator.domain.entity.ExecutionParameter;
+import org.mobilitydata.gtfsvalidator.usecase.port.ExecutionParameterRepository;
+import org.mockito.ArgumentMatchers;
+import org.mockito.InOrder;
+
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
+
+import static org.mockito.Mockito.*;
+
+class ParseAllExecutionParameterTest {
+
+    @Test
+    public void allExecutionParameterFromCommandLineShouldBeParsedAndAddedToRepoAsExecutionParameterEntities() throws IOException {
```

Comment:
Same as Sean mentioned for `ApacheExecutionParameterParser`, maybe we could uses abrevations for some words : Execution -> Exec, Parameter -> Param, Argument -> Arg. 
Also, maybe a name like `allExecParamToRepoTest` could do the trick and we could add some comments/javadoc to explain what should be the exact behaviour expected for the test.

Reply:
Good point also!